rasa/nlu/components.py

```diff
@@ -208,18 +209,36 @@ def validate_pipeline(pipeline: List["Component"]) -> None:
     validate_required_components(pipeline)
 
 
-def any_components_in_pipeline(components: Iterable[Text], pipeline: List["Component"]):
+def any_components_in_pipeline(
+    components: Iterable[Text], pipeline: List["Component"]
+) -> bool:
     """Check if any of the provided components are listed in the pipeline.
 
     Args:
-        components: A list of :class:`rasa.nlu.components.Component`s to check.
+        components: A list of str of component class names to check
```

Comment:
optional: I prefer to not duplicate type in type annotation and docstrings.
```suggestion
        components: Component class names to check.
```